src/shared/prerendered-app/Intro/SlideOnScroll/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { h, Component, RenderableProps } from 'preact';
```

Comment:
I made this its own component to keep the logic in one place.

I also put the style changes and animation in JS rather than CSS. Not because I have anything against doing it in CSS, but it was such as small amount of style it seemed weird to put it in a different file.

Also, one-off animations are much easier to control in JS IMO.

Reply:
Awesome yeah the JS route is great too